BugSplat.uplugin

```diff
@@ -28,11 +28,8 @@
 		}
 	],
 	"PostBuildSteps": {
-		"Win64": [
-			"call \"$(ProjectDir)\\Plugins\\BugSplat\\Source\\Scripts\\BugSplat.bat\" $(TargetPlatform)"
-		],
 		"Mac": [
```

Comment:
Why did this go away? I believe we need this to upload Win64 symbols for Desktop.